decidim-dev/config/rubocop/capybara.yml

```diff
@@ -3,9 +3,10 @@ require: rubocop-capybara
 Capybara:
   Enabled: true
   DocumentationBaseURL: https://docs.rubocop.org/rubocop-capybara
-  Include: &1
+  Include:
     - "**/spec/**/*.rb"
 
 Capybara/RSpec:
-  Enabled: false
-  Include: *1
+  Enabled: true
+  Include:
+    - "**/spec/**/*.rb"
```

Comment:
```suggestion
```

Reply:
This is what I tried to tell with my [other comment ](https://github.com/decidim/decidim/pull/12531#discussion_r1504109690)